src/nti/transactions/transactions.py

```diff
@@ -237,6 +253,16 @@ def do(*args, **kwargs):
     transaction.get().join(result)
     return result
 
+def do_last(*args, **kwargs):
```

Comment:
`OrderedNearEndObjectDataManager` and `do_near_end`? (I suppose even `NearLast` would get the point across.)